packages/workflows/src/api/doc/IParamsCreateWorkflow.ts

```diff
@@ -0,0 +1,118 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession } from "@zowe/imperative";
+import { accessT } from "./ICreateWorkflow";
+
+
+/**
+ * Parameters for the createWorkflow method.
+ *
+ * @export
+ * @interface IParamsCreateWorkflow
+ */
+export interface IParamsCreateWorkflow {
+    /**
+     * z/OSMF session for the workflow creation.
+     *
+     * @type {AbstractSession}
+     * @memberof IParamsCreateWorkflow
+     */
+    session: AbstractSession;
+
+    /**
+     * Workflow title.
+     *
+     * @type {string}
+     * @memberof IParamsCreateWorkflow
+     */
+    WorkflowName: string;
```

Comment:
I noticed that some of these parameters on the interface are in UpperCamelCase, and some are in lowerCamelCase. I think we would want to be consistent in which one we are using, and most other interfaces seem to use lowerCamelCase.